src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/measure/ActionMeter.kt

```diff
@@ -40,7 +34,7 @@ class ActionMeter private constructor(
             "\noutput = output" +
             "\n)" +
             "\n.clock(clock)" +
-            "\n.performanceTimeline(w3cPerformanceTimeline)" +
+            "\n.postMetricHook(DrillDownHook(w3cPerformanceTimeline))" +
```

Comment:
You can add `ReplaceWith` `imports` param to cover `DrillDownHook`

Reply:
`postMetricHook` no longer exists, right?

Reply:
The code here is outdated.